src/Table/index.js

```diff
@@ -134,7 +139,7 @@ class Table extends PureComponent {
             <TableHeaderCell
               key={`${title}-${columnIndex}`}
               align={align}
-              onClick={() => this.handleSortingClick(columnIndex)}
+              onClick={() => this.handleSortingClick(columnIndex, sortable)}
```

Comment:
Is it better to do something like `sortable ? this.handleSortingClick(columnIndex) : undefined` I don't really know

Reply:
I used your advice `sortable ? this.handleSortingClick(columnIndex) : undefined` i think is more readable